File: fizz_buzz.rb
class Basic

  def fizz_buzz (lower = 1, upper = 100)

    (lower..upper).each { |num|
      if num % 5 == 0 and num % 10 == 0 then puts "FizzBuzz"
      elsif num % 5 == 0 then puts "Fizz"
      elsif num % 10 == 0 then puts "Buzz"
      else puts num
      end
    }
  end
end

args = [1,1000]
a = Basic.new
a.fizz_buzz(*args)

File: python/fizz_buzz.py
# Fizz Buzz


def generate_output(start, end):
  for num in xrange(start - 1, end + 1):
  	if num % 3 == 0 and num % 5 == 0:
  		print "FizzBuzz"
  	elif num % 3 == 0:
  		print "Fizz"
  	elif num % 5 == 0:
  		print "Buzz"
  	else:
  	  print num 



if __name__  == "__main__":
  generate_output(1,100)

